tests/core/mempool/test_mempool_manager.py

```diff
@@ -351,7 +358,7 @@ def make_bundle_spends_map_and_fee(
     spend_bundle: SpendBundle, npc_result: NPCResult
 ) -> Tuple[Dict[bytes32, BundleCoinSpend], uint64]:
     bundle_coin_spends: Dict[bytes32, BundleCoinSpend] = {}
```

Comment:
what is a `BundleCoinSpend`?

Reply:
It's a data structure that bridges between `CoinSpend` and `Spend` so that a `coin_id` can give us cheap access to both `coin_spend` and `additions` (as well as eligibility info). It was introduced with the deduplication feature.